chains/bsc.json

```diff
@@ -17,7 +17,23 @@
   "providers": [
     {
       "alias": "default",
-      "rpcUrl": "https://rpc.ankr.com/bsc"
+      "rpcUrl": "https://bsc-dataseed1.binance.org/"
```

Comment:
FYI @andreogle this won't work for Airnode (and ChainAPI) because this provider doesn't support eth_getLogs. Airnode requirements change between versions and config parameters so that's generally ill-defined anyway. Here we're focusing on contract deployments, Airseeker, frontend usage, etc.